src/function_spaces.jl

```diff
@@ -186,11 +181,80 @@ function reference_coordinates(fs::Lagrange{2, RefCube, 1})
             Vec{2, Float64}((-1.0,  1.0,)))
 end
 
+##################################
+# Lagrange dim 2 RefCube order 2 #
+##################################
+n_basefunctions(::Lagrange{2, RefCube, 2}) = 8
 
-###################################
-# Lagrange dim 2 RefTetrahedron order 1 #
-###################################
+function value!(fs::Lagrange{2, RefCube, 2}, N::Vector, ξ::Vec{2})
```

Comment:
Isn't this what I call serendipity below? A la grange element would have 9 nodes?